tests/discover/libraries/data/plan.fmf

```diff
@@ -68,3 +71,7 @@ execute:
             summary: "Missing reference"
             discover+:
                 test: missing/reference
+/querying:
+    summary: "Many tests requiring same rpm-only library"
+    discover+:
+        test: querying
```

Comment:
There are no `querying` tests added in the pull request, thus the test fails:

```
:: [ 10:40:15 ] :: [  BEGIN   ] :: Running 'tmt run --id /tmp/tmp.ZFAtydDQJT/querying plan --name querying discover'
/tmp/tmp.ZFAtydDQJT/querying

/plan/querying
    discover
        how: fmf
        directory: /home/psss/git/tmt/tests/discover/libraries/data
        tests: querying
        summary: 0 tests selected
        warning: No tests found, finishing plan.
:: [ 10:40:16 ] :: [   PASS   ] :: Command 'tmt run --id /tmp/tmp.ZFAtydDQJT/querying plan --name querying discover' (Expected 0, got 0)
:: [ 10:40:16 ] :: [   FAIL   ] :: File '/tmp/tmp.ZFAtydDQJT/querying/log.txt' should contain 'git clone .*beakerlib/FOOBAR' 
```

Perhaps a forgotten `git add`?

Reply:
Indeed, fixed in fcf8bce4